apps/researcher/src/messages/nl/messages.json

```diff
@@ -111,6 +111,7 @@
     "birthPlacesFilter": "Geboorteplaatsen",
     "deathYearsFilter": "Overlijdensjaren",
     "deathPlacesFilter": "Overlijdensplaatsen",
+    "publishersFilter": "Object data providers",
```

Comment:
Is there a Dutch term for 'data providers'?

Reply:
Camiel suggested: "Bronhouders object"

Reply:
I like it!